frontend/src/lib/validation.ts

```diff
@@ -287,3 +290,77 @@ export function validateNoProxy(value: string, t: TFunction) {
     }
     return undefined
 }
+
+// export function validateKubernetesDnsName(value: string, t: TFunction) {
+//     if (value) {
+//         if (value.length > 63) return `${t('validate.kubernetesDnsName.length')}`
+//         for (const char of value) {
+//             if (!lowercaseAlphaNumericCharacters.includes(char) && char !== '-')
+//                 return `${t('validate.kubernetesDnsName.char')}`
+//         }
+//         if (!lowercaseAlphaNumericCharacters.includes(value[0])) return `${t('validate.kubernetesDnsName.startchar')}`
+//         if (!lowercaseAlphaNumericCharacters.includes(value[value.length - 1]))
+//             return `${t('validate.kubernetesDnsName.endchar')}`
+//     }
+//     return undefined
+// }
```

Comment:
Looks like these were already ported over? Can you confirm the logic is the same and remove commented portions from this file if they are not needed?

Reply:
confirmed and removing this commented out validation.  Leaving the additional validations(RFC1123 and RFC 1135) commented in case they are used in the future.